### PR TITLE
Update FE Packages to the Latest Compatible and Stable Version

### PR DESCRIPTION
**Pull request by vahidbazzaz**
In this PR, we are updating the following core libraries to their latest stable major versions: React, Apollo-client, React-table, React-hook-form, Chakra-ui, Chakra-react-select, sentry and react-select.

### CODE REVIEW

**Comment by vahidbazzaz** [comment]
[Update FE Packages to the Latest Compatible and Stable Version](https://trello.com/c/kaNt5ifE)

File: react/package.json [hunk 1]
```diff
@@ -5,79 +5,81 @@
...
-    "@auth0/auth0-react": "^1.10.2",
+    "@auth0/auth0-react": "^1.11.0",
+    "@babel/runtime": "^7.19.0",
     "@chakra-ui/icons": "^1.1.7",
-    "@chakra-ui/react": "^1.6.10",
-    "@emotion/react": "^11",
-    "@emotion/styled": "^11",
-    "@fortawesome/fontawesome-svg-core": "^1.2.36",
+    "@chakra-ui/react": "^1.8.9",
+    "@emotion/react": "^11.10.4",
```
Comment (HaGuesto) [inline react/package.json:12]: figure out what this is for.

File: react/package.json [hunk 2]
```diff
@@ -5,79 +5,81 @@
...
+    "@sentry/integrations": "^7.13.0",
+    "@sentry/react": "^7.13.0",
+    "@sentry/tracing": "^7.13.0",
     "@zxing/browser": "^0.1.1",
     "@zxing/library": "^0.19.1",
-    "chakra-react-select": "^3.3.6",
-    "date-fns": "^2.29.1",
+    "chakra-react-select": "^4.2.2",
+    "date-fns": "^2.29.3",
     "framer-motion": "^6.3.10",
```
Comment (HaGuesto) [inline react/package.json:25]: drop this or figure out where it is needed.

File: react/package.json [hunk 3]
```diff
@@ -5,79 +5,81 @@
...
-    "react-hook-form": "^7.31.3",
+    "react-hook-form": "^7.36.0",
     "react-icons": "^4.4.0",
-    "react-router-dom": "6",
-    "react-scripts": "5.0.1",
-    "react-table": "^7.7.0",
+    "react-router-dom": "^6.4.1",
+    "react-scripts": "^5.0.1",
+    "react-table": "^7.8.0",
     "regenerator-runtime": "^0.13.9",
```
Comment (HaGuesto) [inline react/package.json:36]: to dev Deps

File: react/package.json (hunk 1 above)
Comment (aerinsol) [reply react/package.json:12]: Chakra UI is what @spaudanjo and @flisowna use for React components.

Comment (vahidbazzaz) [reply react/package.json]: I have checked [Chakra UI](https://chakra-ui.com/getting-started) and @emotion/react, @emotion/styled, and framer-motion are its dependencies

**Pull request merged**